src/WebJobs.Script/Host/ScriptHostManager.cs

```diff
@@ -30,6 +32,9 @@ public class ScriptHostManager : IScriptHostEnvironment, IDisposable
         private readonly IScriptHostEnvironment _environment;
         private readonly IDisposable _fileEventSubscription;
         private readonly StructuredLogWriter _structuredLogWriter;
+        private readonly HostPerformanceManager _performanceManager;
+        private readonly Timer _hostHealthCheckTimer;
+        private readonly TimeSpan hostHealthCheckInterval = TimeSpan.FromSeconds(30);
```

Comment:
Might make the interval shorter - thoughts? It's just an environment variable access + some computations

Reply:
Yeah I think this should be more frequent. We're not doing any IO here so it should be really cheap. Maybe 5 seconds?

Reply:
Alright - let's start with 15 seconds.